config/locales/champaign.en.yml

```diff
@@ -243,6 +243,7 @@ en:
     petition:
       petition_description: "Petition Text"
       choose_form_template: "Use a form template"
+      form_template_placeholder: '-- select form template --'
```

Comment:
If you want to keep these dashes, shouldn't you rather add them to the element directly, rather than the translation?